RTC_wireless_sensor_nodes_firmware/Fermentation_WSN/Fermentation_WSN.ino

```diff
@@ -1,15 +1,23 @@
+/*
+ * To make this file universal for all three fermentation tanks
+ * "PH 1: DIP1=OFF, DIP2=OFF" ==>> NodeID:23
+ * "PH 2: DIP1=ON, DIP2=OFF" ==>> NodeID:24
+ * "PH 3: DIP1=OFF, DIP2=ON" ==>> NodeID:25
+ * 
+ */
+  
 #define RF69_COMPAT 1                                                              // Set to 1 if using RFM69CW or 0 is using RFM12B
 #include <JeeLib.h>                                                                      //https://github.com/jcw/jeelib - Tested with JeeLib 3/11/14
 
 boolean debug=1;                                       //Set to 1 to few debug serial output, turning debug off increases battery life
 
 #define RF_freq RF12_433MHZ                 // Frequency of RF12B module can be RF12_433MHZ, RF12_868MHZ or RF12_915MHZ. You should use the one matching the module you have.
-int nodeID = 21;                               // EmonTH temperature RFM12B node ID - should be unique on network
-const int networkGroup = 210;                // EmonTH RFM12B wireless network group - needs to be same as emonBase and emonGLCD
-//uint32_t networkGroup = 3421749817;    //numeric value from ansible rmc_key
+int nodeID = 23;                               // EmonTH temperature RFM12B node ID - should be unique on network
+//const int networkGroup = 210;                // EmonTH RFM12B wireless network group - needs to be same as emonBase and emonGLCD
+uint32_t networkGroup = 3421749817;    //numeric value from ansible rmc_key
                                                                       
 
-const int time_between_readings= 2;                                   // in minutes
+const int time_between_readings= 1;                                   // in minutes
```

Comment:
No need to change that,
Not sure how i changed that, am going to keep 1 min interval